pkgs/top-level/python-packages.nix

```diff
@@ -6910,6 +6910,8 @@ self: super: with self; {
 
   nbmerge = callPackage ../development/python-modules/nbmerge { };
 
+  nbqa = callPackage ../development/python-modules/nbqa { };
```

Comment:
Since the package has no library surface it cannot stay in python-packages.

Reply:
All done.